mobile/mobile.go

```diff
@@ -121,7 +122,8 @@ func MigrateRepo(config *MigrateConfig) error {
 // Create a gomobile compatible wrapper around Textile
 func NewTextile(config *RunConfig, messenger Messenger) (*Mobile, error) {
 	node, err := core.NewTextile(core.RunConfig{
-		RepoPath: config.RepoPath,
+		RepoPath:  config.RepoPath,
+		LogLevels: map[string]string{"*": config.LogLevel},
```

Comment:
I think need more control here... gonna be super noisy. Could you allow something like `config.LogLevels` plural, as JSON (name: level)

Reply:
Yup for sure, that's what we have on desktop etc.